terra-workspace-manager/variables.tf

```diff
@@ -57,6 +58,12 @@ variable "workspace_project_folder_id" {
   default = ""
 }
 
+variable "workspace_folder_ids_from_rbs" {
+  type        = list(string)
+  description = "List of folder ids WSM will need to be able to access. These folders contain projects created by RBS. Folders are created outside of WSM."
```

Comment:
I don't think the WM terraform should say anything about where its folders come from - I don't think it should care. So I think these inputs should just describe what happens to these folder ids, not that they're created by RBS. (And in the AoU, we're talking about the folder not being created by RBS).

Reply:
Agree, updated.